playbooks/openstack/advanced-configuration.md

```diff
@@ -274,6 +274,38 @@ openshift_openstack_cluster_node_labels:
     mylabel: myvalue
 ```
 
+`openshift_openstack_cloud_init_runcmd` allows users to execute
+shell commands via cloud-init for all of the created Nova servers in
+the Heat stack, before they are available for SSH connections.
+Note that you should better off using custom ansible playbooks whenever
```

Comment:
"Note that you should use custom..."

Reply:
I never get those "<a verb> off" right :(